test/extensions/transport_sockets/tls/context_impl_test.cc

```diff
@@ -1805,16 +1805,28 @@ TEST_F(ServerContextConfigImplTest, TlsCertificatesAndSdsConfig) {
       "SDS and non-SDS TLS certificates may not be mixed in server contexts");
 }
 
-TEST_F(ServerContextConfigImplTest, MultiSdsConfig) {
+TEST_F(ServerContextConfigImplTest, SdsConfigNoName) {
```

Comment:
The old test does not make sense. The validation failure is caused by not setting name instread of dual configs.